modules/nw-ingress-controller-configuration-gcp-global-access.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-gcp-global-access_{context}"]
+= Configuring Global Access parameters on GCP
+As a cluster administrator with an internal load balancer on GCP, you can specify the global access option. Clients in any region within the same VPC network as the load balancer can reach the workloads running on your cluster.
+
+For more information, please reference the GCP documentation for link:https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#global_access[Global access].
+
+.Prerequisites
+
+* You deployed an {product-title} cluster on GCP infrastructure.
+* You created an ingress controller with internal load balancing.
```

Comment:
Maybe link to https://docs.openshift.com/container-platform/4.7/networking/ingress-operator.html#nw-ingress-setting-internal-lb_configuring-ingress here?

Reply:
Unfortunately, I'm unable to link right here, but I can put this link in the additional resources at the bottom!

Reply:
@sgreene570 Since we're unable to do internal linking at the moment, I tried to add a few more introductory details to give this topic  more context.

Reply:
sounds good.